code/modules/jobs/job_types/chaplain/chaplain_nullrod.dm

```diff
@@ -27,7 +27,7 @@
 
 /obj/item/nullrod/Initialize(mapload)
 	. = ..()
-	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE|MAGIC_RESISTANCE_HOLY)
+	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE|MAGIC_RESISTANCE_MIND|MAGIC_RESISTANCE_HOLY)
```

Comment:
I really dislike this, the chaplain should not become the premiere anti-abductor crewmember

Reply:
Oh, I hadn't even noticed this change, this doesn't sit right with me either. I think at this point you may as well just have Moon spells be affected by regular antimagic again rather than expand the chaplain's defenses to stuff outside of magic-related things.

Reply:
gonna remove mind antimagic